gradle.properties

```diff
@@ -1,5 +1,5 @@
 ideaBranch=182
-ideaVersion=182.2574.2
+ideaVersion=182.3684.2
```

Comment:
This shouldn't be merged into the idea-all branch, as that has it's own version ranges